src/lando/headless_api/tests/test_automation_api.py

```diff
@@ -28,14 +28,35 @@
 from lando.pushlog.models import Push
 
 
+@pytest.fixture
+def automation_job():
+    """Create an automation job from the specified actions."""
+
+    def create_automation_job(actions: list[dict], **job_args):
```

Comment:
why not add return type here also since there are other type hints?

Reply:
Good call, added.